docs/permissions/data-sandboxes-draft.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Data sandboxes
+---
+
+# Data sandboxes
+
+{% include plans-blockquote.html feature="Data sandboxes" %}
+
+Data sandboxes let you give granular permissions to rows and columns for different groups of people.
+
+You can think of a data sandbox as a permissions "container" that includes:
+
+- The edited version of a table in your database (for example, with some rows or columns removed).
+- The [group](../people-and-groups/managing.md#groups) of people who should see the edited version of the table instead of the original table, everywhere that the table is used in Metabase.\*
+
+You can define one data sandbox (that is, one permission set) for each table and group pairing in your Metabase.
+
+\* SQL questions cannot be sandboxed. See [Preventing data sandboxing permissions conflicts](#preventing-data-sandboxing-permissions-conflicts).
+
+## Types of data sandboxes
+
+For each group and table pairing, you can choose between:
+
+- A [row-limiting data sandbox](#row-limiting-data-sandboxes), to hide rows from a table based on a user attribute.
+- An [advanced data sandbox](#advanced-data-sandboxes), to display a custom query result in place of a table (the attribute-based filter is optional).
+
+### Row-limiting data sandboxes
+
+A row-limiting sandbox displays the result of a filtered table. Groups assigned to a row-limiting sandbox will see a version of the table with some `Filter = Value` applied, instead of the original table.
+
+For example, you can create a row-limiting sandbox to filter the Accounts table so that one person sees the rows where `Plan = Basic`, and another person only sees the rows where `Plan = Premium`. 
+
+The data sandbox will dynamically set the filter value to "Basic" or "Premium" to hide different rows for each person based on their [user attributes](../people-and-groups/managing.md#adding-user-attributes).
+
+### Advanced data sandboxes
```

Comment:
I thought about this more. I think that the "advanced sandbox" is more like a "custom sandbox" because you can put anything in the saved SQL question.

Restricting columns will be the most common use case, but you can do a lot of other useful security things. Like displaying a hashed column instead of an original column. Or aggregating a survey results table and displaying responses where the count is > 5 (common when dealing with PII).

Basically I'm thinking of putting this line into the planned Learn tutorials:
> [Create a custom sandbox]() for Group A so that they can only see columns X and rows Y.